chart/kubeapps/Chart.yaml

```diff
@@ -1,19 +1,19 @@
 apiVersion: v1
 name: kubeapps
-version: 0.3.2
+version: 0.3.3
 appVersion: DEVEL
 description: Kubeapps is a dashboard for your Kubernetes cluster that makes it easy to deploy and manage applications in your cluster using Helm
 icon: https://raw.githubusercontent.com/kubeapps/kubeapps/master/docs/img/logo.png
 keywords:
-- helm
-- dashboard
-- service catalog
-- deployment
+  - helm
```

Comment:
this new indentation was automatically generated by the vscode editor

Reply:
It doesn't do that for me, do you have some sort of YAML lint extension or something? The rest of the chart (e.g. values.yaml) doesn't indent lists twice so we should keep it consistent with that for now.

Reply:
I don't, it's the normal `editor.formatOnSave` feature. I will disable it 😞